docs_src/security/Ch-CORS-Settings.md

```diff
@@ -67,3 +67,39 @@ There is not a way, when using the API gateway, to turn CORS on for one microser
 !!! note
     The name of the configuration sections and environment variable overrides are intentionally different than the API gateway section, in alignment with the guidance that CORS should be enabled at the microservice level or the API gateway level, but not both.  Thus, the security-enabled overrides are accomplished with `EDGEX_SERVICE_CORSCONFIGURATION_*` overrides, and the no-security overrides with `SERVICE_CORSCONFIGURATION_*`.
 
+
+### Enabling CORS for the EdgeX Snap (via API gateway)
+
+To enable CORS support in the API gateway in the EdgeX Snap, a slightly different procedure is required.
+
+First, we need to override the `EDGEX_SERVICE_CORSCONFIGURATION_*` environment variables like was done in Docker.
+However, we need to override this in the `security-bootstrapper-nginx` service.
+This service runs before `nginx.service` to write the NGINX configuration file.
+Then, we need to restart the `security-bootstrapper-nginx` service to generate a new configuration,
+and also restart `nginx.service` to put the new configuration into effect.
```

Comment:
```suggestion
If started prior to this configuration, restart the `security-bootstrapper-nginx` service to generate a new configuration,
and also restart `nginx` to put the new configuration into effect.
Otherwise, start the services as usual.
```